archive/html/OpenAI/Openai-codex-agents-multiagent-best-practices.html

```
@@ -0,0 +1,1622 @@
+          margin-left: 0;
+          padding: 88px 20px 70px;
+        }
+        ul.ref-list {
+          grid-template-columns: minmax(0, 1fr);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="layout">
+      <button class="sidebar-toggle" id="sidebarToggle" aria-label="メニュー">☰</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)